package.json

```diff
@@ -30,9 +30,11 @@
   },
   "dependencies": {
     "@babel/plugin-transform-exponentiation-operator": "^7.16.0",
+    "@dfinity/agent": "0.9.3",
     "@dfinity/candid": "0.9.3",
     "@dfinity/principal": "^0.9.3",
     "@hookform/error-message": "^2.0.0",
+    "@psychedelic/dab-js": "^1.4.6",
```

Comment:
install it again without the `^` in the version so we don't have future conflicts with 0.2.0